lib/Computer/computer.js

```diff
@@ -0,0 +1,17 @@
+export class Computer {
+    constructor(token1, token2) {
+        this.player1 = token1,
+        this.player2 = token2,
+        this.currentPlayer = this.player1
+    };
+
+    // public getCurrentPlayerTokenObj(): Token
+    getCurrentPlayerTokenObj() {
+        return this.currentPlayer;   
+    };
+
+    // public alternatePlayers(): void
+    alternatePlayers() {
+        this.currentPlayer === this.player1 ? this.currentPlayer = this.player2 : this.currentPlayer = this.player1;
+    };
+};
```

Comment:
🥕 Food for thought: 

What is the responsibility of the `Computer` class? Is it to represent a `Computer` player and have it to what a person would do (i.e. pick a place on the board to place a token)? 

Is its responsibility to delegate what players do in a game (i.e. who goes first and determining whose turn it is)?